### ISSUE TITLE
Implement XEP-0184: Message Delivery Receipts

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vacuum-im/vacuum-im#537]
```
XEP-0184: Message Delivery Receipts is a very useful feature. I think, it 
should be implemented for sure.

XEP-0184: Отчёты о Доставке Сообщений - весьма 
полезная фича. Я считаю, что её обязательно 
следует реализовать.
```

Original issue reported on code.google.com by `yagimo...@gmail.com` on 6 Dec 2011 at 9:00
- Blocked on: #538
- Merged into: #105
